packages/ag-charts-community/src/chart/factory/expectedModules.ts

```diff
@@ -0,0 +1,101 @@
+    { type: 'series', name: 'funnel', chartType: 'cartesian', enterprise: true },
+    { type: 'series', name: 'ohlc', chartType: 'cartesian', enterprise: true },
+    { type: 'series', name: 'heatmap', chartType: 'cartesian', enterprise: true },
+    { type: 'series', name: 'histogram', chartType: 'cartesian' /*, enterprise: true*/ },
+    { type: 'series', name: 'range-area', chartType: 'cartesian', enterprise: true },
+    { type: 'series', name: 'range-bar', chartType: 'cartesian', enterprise: true },
+    { type: 'series', name: 'waterfall', chartType: 'cartesian', enterprise: true },
+    { type: 'series', name: 'nightingale', chartType: 'polar', enterprise: true },
+    { type: 'series', name: 'radar-area', chartType: 'polar', enterprise: true },
+    { type: 'series', name: 'radar-line', chartType: 'polar', enterprise: true },
+    { type: 'series', name: 'radial-bar', chartType: 'polar', enterprise: true },
+    { type: 'series', name: 'radial-column', chartType: 'polar', enterprise: true },
+    { type: 'series', name: 'map-shape', chartType: 'topology', enterprise: true },
+    { type: 'series', name: 'map-line', chartType: 'topology', enterprise: true },
+    { type: 'series', name: 'map-marker', chartType: 'topology', enterprise: true },
+    { type: 'series', name: 'map-shape-background', chartType: 'topology', enterprise: true },
+    { type: 'series', name: 'map-line-background', chartType: 'topology', enterprise: true },
+    { type: 'series', name: 'pyramid', chartType: 'standalone', enterprise: true },
+    { type: 'series', name: 'linear-gauge', chartType: 'standalone', enterprise: true },
+    { type: 'series', name: 'radial-gauge', chartType: 'standalone', enterprise: true },
+    { type: 'series', name: 'sunburst', chartType: 'standalone', enterprise: true },
+    { type: 'series', name: 'treemap', chartType: 'standalone', enterprise: true },
+    { type: 'series', name: 'chord', chartType: 'standalone', enterprise: true },
+    { type: 'series', name: 'sankey', chartType: 'standalone', enterprise: true },
+
+    // Plugins
+    { type: 'plugin', name: 'animation', enterprise: true },
+    { type: 'plugin', name: 'annotations', chartType: 'cartesian', enterprise: true },
+    { type: 'plugin', name: 'background', enterprise: true, removable: false },
+    { type: 'plugin', name: 'foreground', enterprise: true },
+    { type: 'plugin', name: 'chartToolbar', chartType: 'cartesian', enterprise: true },
+    { type: 'plugin', name: 'contextMenu', enterprise: true },
+    { type: 'plugin', name: 'statusBar', chartType: 'cartesian', enterprise: true },
+    { type: 'plugin', name: 'dataSource', enterprise: true },
+    { type: 'plugin', name: 'sync', chartType: 'cartesian', enterprise: true },
+    { type: 'plugin', name: 'ranges', chartType: 'cartesian', enterprise: true },
+    {
+        type: 'plugin',
+        name: 'zoom',
+        // chartTypes: ['cartesian', 'topology']
```

Comment:
is this needed? zoom is still functionally limited to these chart types I believe